### ISSUE TITLE
Update ux documentation

### ISSUE THREAD

**Issue by codecakes** [Xcov19/mhc-fe#12]
All UX documentation should be updated here: [https://xcov19.dev](https://xcov19.dev/documentation/#design-style-guide)

**Comment by rejuvinatez** [comment]
What is UX Documentation for?

**Comment by emanuelcaaju** [comment]
@rejuvinatez We will bring the components that will be used in the project into a design system, but today we are introducing figma tokens to the project, as it will still become a complement to this documentation

**Comment by AlexanderAlzona** [comment]
@rejuvinatez we would use this as a reference of standards for use, further in the development life cycle.  We should keep in mind to keep the pervious iterations of the UX documentations for comparison later on.  So if we do decide to update the documentation, keep the old version somewhere for a reference component later on. i.e. version tracking in the long run.

**Comment by codecakes** [comment]
Version tracking needs to be a separate feature request. it is not in the scope of this issue

**Comment by AlexanderAlzona** [comment]
@codecakes I agree, the scope of the issue is keeping the old version when this project was "Xcov19", then looking at this project as it is now "My Health Connect" and implimenting it's own documentation.

**Comment by codecakes** [comment]
> @codecakes I agree, the scope of the issue is keeping the old version when this project was "Xcov19", then looking at this project as it is now "My Health Connect" and implimenting it's own documentation.

By mentioning "versions", "old", "new", projects, you are introducing scope creep here. This is just the UX documentation. All projects here are indeed under the "Project XCoV19" and this is its UX documentation, which is mostly here: https://xcov19.dev/documentation/#design-style-guide

And anything in addition should be under https://xcov19.dev/documentation/

**Comment by Shirinzaboli** [comment]
@emanuelcaaju please hold a live session with one of the UX developers on how to use Figma token for the project then everyone can try on their own.
